.about.yml

```diff
@@ -0,0 +1,25 @@
+---
+project: "Analytics.USA.gov"
+name: dap
+github:
+- gsa/analytics.usa.gov
+- 18f/analytics-reporter
+description: A lightweight dashboard that shows web traffic across the .gov domain.  
+partners:
+- General Services Administration
+stage: beta
+milestones:
+- 'May 2012: Digital Government Strategy calls for a government-wide analytics program'
+- 'May 2013: Digital Analytics Program announced'
+- 'August 2014: Full Digital Analytics Program access expanded to all federal staff at participating agencies'
+- 'March 2015: Analytics.USA.gov announced'
+contact:
+- dap@gsa.gov
+stack: jekyll, bourbon, neat, d3, node, analytics-reporter
+team: gray, eric, leah, shawn
```

Comment:
The team has changed since then, and now includes Gabriel and Julia and potentially others.